README.md

```diff
@@ -219,6 +219,25 @@ A Local Streem uses the device's camera, and opens up an AR experience with our
 
 Note: Due to an issue with ARKit, you cannot start a Local Streem from a view using the camera https://forums.developer.apple.com/message/411888#411888
 
+## Pro Implementation
+The above is sufficient to implement the Customer side of streems (the caller). If you want to implement the Pro side of streems (the callee) the following will get you started.
+
+### Logging In
+Before you can start doing Pro things you'll need to login as a Pro. A user which is created with Streem can be designated as a Pro through the admin portal. Once designated, the user should login using their username and password:
+
+```swift
+    Streem.sharedInstance.login(
+        companyCode: "My Company",
```

Comment:
Shouldn't this be a company code rather than a company name? We don't allow spaces or upper-case characters in company codes.

Reply:
👍